### PR TITLE
Add low reliability annotations

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard1**
## Summary

As the title states:

image

### Governance

- [ ] Documentation is added
- [ ] Test cases are added or updated
- [ ] I've read the contributing document https://github.com/minvws/.github/blob/master/CONTRIBUTING.md
- [ ] I've read and understand the Code of Conduct https://github.com/minvws/.github/blob/master/CODE_OF_CONDUCT.md
- [ ] I understand that any contributions or suggestions I made may make it into the actual code. I've read the License
      https://github.com/minvws/nl-covid19-notification-app-coordination/blob/master/LICENSE.txt
      and the contributor license agreement https://github.com/minvws/nl-covid19-notification-app-coordination/blob/master/CLA.md

### CODE REVIEW

File: packages/app/src/domain/international/variants-stacked-area-tile/logic/use-unreliable-data-annotations.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        .filter((x) => isDefined(x.is_reliable) && x.is_reliable === false)
+        .map<TimespanAnnotationConfig>((x) => ({
+          start: x.date_start_unix,
+          end: x.date_end_unix,
+          label,
+          fill: 'dotted',
+        })),
+    [values, label]
+  );
+}
```
Comment (VWSCoronaDashboard10) [inline packages/app/src/domain/international/variants-stacked-area-tile/logic/use-unreliable-data-annotations.tsx:22]: I think it's perfectly fine to write this useMemo inside the `VariantsStackedAreaTile`. Having it inside another file makes it harder to read/understand IMHO.

File: packages/app/src/domain/international/variants-stacked-area-tile/logic/use-unreliable-data-annotations.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import { VariantChartValue } from '~/static-props/variants/get-variant-chart-data';
+
+export function useUnreliableDataAnnotations(
+  values: VariantChartValue[],
+  label: string
+) {
+  return useMemo(
+    () =>
+      values
+        .filter((x) => isDefined(x.is_reliable) && x.is_reliable === false)
```
Comment (VWSCoronaDashboard10) [inline packages/app/src/domain/international/variants-stacked-area-tile/logic/use-unreliable-data-annotations.tsx:13]: Does TS accept this? It's practically the same:
```suggestion
        .filter((x) => x.is_reliable === false)
```

File: packages/app/src/static-props/variants/get-international-variant-chart-data.ts [hunk 3]
```diff
@@ -62,6 +65,12 @@ export function getVariantChartData(variants: InVariants | undefined) {
         if (isDefined(otherItem) && isPresent(otherItem.percentage)) {
           total += otherItem.percentage;
           item[`${variantOfConcern.name}_percentage`] = otherItem.percentage;
+          if (
+            isDefined(otherItem.is_reliable) &&
+            otherItem.is_reliable === false
+          ) {
```
Comment (VWSCoronaDashboard10) [inline packages/app/src/static-props/variants/get-international-variant-chart-data.ts:71]: same here:
```suggestion
          if (otherItem.is_reliable === false) {
```

**Review by VWSCoronaDashboard10 — APPROVED** [review]

File: packages/app/src/domain/international/variants-stacked-area-tile/logic/use-unreliable-data-annotations.tsx (hunk 2 above)
Comment (VWSCoronaDashboard1) [reply packages/app/src/domain/international/variants-stacked-area-tile/logic/use-unreliable-data-annotations.tsx:13]: True, but in theory, data could be passed in without an `is_reliable` property (this type is shared between national and into), and if the is_reliable is undefined, it is assumed to be reliable.

Comment (VWSCoronaDashboard10) [reply packages/app/src/domain/international/variants-stacked-area-tile/logic/use-unreliable-data-annotations.tsx]: Yeah but:
```tsx
({ is_reliable: false     }.is_reliable === false) === true
({ is_reliable: undefined }.is_reliable === false) === false
({                        }.is_reliable === false) === false

Comment (VWSCoronaDashboard1) [reply packages/app/src/domain/international/variants-stacked-area-tile/logic/use-unreliable-data-annotations.tsx]: haha, oh wait a sec, I had my boolean logic backwards, you're absolutely right haha

**Review by VWSCoronaDashboard8 — APPROVED** [review]

**Pull request merged**